src/TextField/TextField.jsx

```diff
@@ -304,7 +304,7 @@ const TextField = React.createClass({
         height: '100%',
         border: 'none',
         outline: 'none',
-        backgroundColor: 'transparent',
+        backgroundColor: 'rgba(0,0,0,0)',
```

Comment:
We define `transparent` as `rgba(0,0,0,0` in `material-ui/lib/styles/colors`, but it's only used in one place.

@oliviertassinari  what's preferred here?